rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -169,10 +169,30 @@ components:
       type: string
       minLength: 1
       maxLength: 256
-    Metacard:
+    Metadatas:
       description: >
-        A metacard.xml attachment that is sent along with a product to enhance the product's discoverability. The
-        current maximum is the character equivalent of 10GB.
+        A list of metadata objects that a client can pass along with a product to enrich indexing and query
+        capabilities.
+      type: array
+      items:
+        $ref: '#/components/schemas/Metadata'
+    Metadata:
+      description: An object that contains the metadata type and the metadata file.
+      type: object
+      properties:
+        metadataType:
```

Comment:
✏️ I would remove `metadata` from the name of these 2 fields since they are part of the `Metadata` schema object to start with